.travis.yml

```diff
@@ -33,7 +33,7 @@ before_install:
 install:
 
   - if [[ $MINIMAL_ENV == 'False' ]] ; then
-      DEPS="pip numpy scipy matplotlib>=2.0.2 ipython h5py sympy scikit-learn dill natsort setuptools scikit-image cython lxml ipyparallel dask traits traitsui";
+      DEPS="pip numpy scipy matplotlib>=2.0.2 ipython h5py sympy scikit-learn dill natsort setuptools scikit-image cython lxml ipyparallel dask traits traitsui blosc mrcz";
```

Comment:
These are the dependencies installed from the conda-forge channel. I cann't find any repo for the mrcz, so I guess it would be better to remove mrcz from here and leave pip install it automatically from pypi when hyperspy is installed.